### ISSUE TITLE
Licence

### ISSUE THREAD

**Issue by calumk** [xenova/Draggable#1]
Hi, I Assume this is MIT License?

**Comment by xenova** [comment]
Hey 👋  The code is quite old (and not very good 😅), but feel free to edit/use/modify it however you want!

**Comment by calumk** [comment]
Yep, I'm basically fully re-writing it as a module but just wanted the go-ahead 

Thanks!
